ngx_http_sxg_filter_module.c

```diff
@@ -447,7 +445,8 @@ static bool copy_buffer_to_sxg_buffer(const ngx_http_request_t* req,
         const size_t buffer_tail = buf->size;
         if (buffer_tail + copy_size > limit) {
           ngx_log_error(NGX_LOG_ERR, req->connection->log, 0,
-                        "Too large buffer size required: %d bytes",
+                        "nginx-sxg-module: too large buffer size required "
```

Comment:
If you call `ngx_log_init("nginx-sxg-module")` during initialization does it centralize this prefix?

Reply:
Unfortunately, No.
`ngx_log_init(prefix)` modifies [global variable](
https://github.com/nginx/nginx/blob/master/src/core/ngx_log.c#L317-L326) and seems to be a function for application wide configuration with [command line flag](https://github.com/nginx/nginx/blob/master/src/core/nginx.c#L790).

Reply:
`ngx_log_error` is a macro which wraps `ngx_log_error_core`.
I'm not sure to re-wrap this with another (prefix appending) macro helps readability.